curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property controls text size?
 
 #### --distractors--
 
-Placeholder distractor 1
+text-size
 
 ---
 
-Placeholder distractor 2
+size-font
 
 ---
 
-Placeholder distractor 3
+text-style
 
 #### --answer--
 
-Placeholder answer
+font-size
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the opacity property do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changes text size
 
 ---
 
-Placeholder distractor 2
+Adds a border
 
 ---
 
-Placeholder distractor 3
+Sets the element's position
 
 #### --answer--
 
-Placeholder answer
+Controls element transparency
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide content from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the flex property used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing text color
 
 ---
 
-Placeholder distractor 2
+Adjusting font size
 
 ---
 
-Placeholder distractor 3
+Adding animations
 
 #### --answer--
 
-Placeholder answer
+Creating responsive layouts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute helps improve accessibility for links?
 
 #### --distractors--
 
-Placeholder distractor 1
+href
 
 ---
 
-Placeholder distractor 2
+target
 
 ---
 
-Placeholder distractor 3
+class
 
 #### --answer--
 
-Placeholder answer
+title
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does CSS stand for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Creative Style Sheets
 
 ---
 
-Placeholder distractor 2
+Colorful Style Sheets
 
 ---
 
-Placeholder distractor 3
+Computer Style Sheets
 
 #### --answer--
 
-Placeholder answer
+Cascading Style Sheets
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for graphics?
 
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+3:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which feature of Chrome helps in accessibility audits?
```

Comment:
Well, to be honest, I'm a profession accessibility auditor and I never use Lighthouse, and I always use the developer tools, so I would get this answer wrong. There are also many extensions that I use for accessibility testing. I would probably change the wording of this question to something like "What automated testing tool is provided by Chrome to aid in accessibility testing?"